config/initializers/i18n.rb

```diff
@@ -19,6 +19,15 @@
 
 I18n.available_locales = %i[en fr zh-CN de ja ru]
 
+# Here are the locales we will *automatically* switch to.
+# This *may* be the same as I18n.available_locales, but if a locale's
+# translation isn't ready we should probably omit it here.
+Rails.application.config.automatic_locales = %i[en fr zh-CN de]
```

Comment:
All of the translations are sufficiently far along that they are better than English as a default.
```ruby
Rails.application.config.automatic_locales = I18n.available_locales
```